app/util/k8s/dcapt-snapshots.json

```diff
@@ -163,8 +163,8 @@
             "size": "small",
             "snapshots": [
               {
-                "us-east-2": "arn:aws:rds:us-east-2:585036043680:snapshot:dcapt-jsm-small-5-12-8",
-                "us-east-1": "arn:aws:rds:us-east-1:585036043680:snapshot:dcapt-jsm-small-5-12-8"
+                "us-east-2": "arn:aws:rds:us-east-2:585036043680:snapshot:dcapt-jsm-small-5-12-8-fixed",
```

Comment:
As discussed, let's try creating new snapshots with the same name.
If not possible, need to create new snapshots for a new version.